errors.go

```diff
@@ -3,8 +3,14 @@ package ngrok
 import (
 	"fmt"
 	"net/url"
+	"regexp"
 )
 
+type ngrokError interface {
```

Comment:
Since this is unexported and unused in the codebase, this actually doesn't have any benefit, right?

Reply:
Yeah, definitely my bad. Will make it exported.